pkg/didcomm/dispatcher/outbound.go

```diff
@@ -40,6 +40,11 @@ func NewOutbound(prov provider) *OutboundDispatcher {
 	}
 }
 
+// SendToDID msg
+func (o *OutboundDispatcher) SendToDID(msg interface{}, myDID, theirDID string) error {
+	return nil
```

Comment:
This will need to be implemented before BDD tests will work.

Reply:
Yeah, I know. I just added this interface to correct the logic